tests/node/rpc/structs/RoofStatusTests.js

```diff
@@ -0,0 +1,43 @@
+const SDL = require('../../../config.js').node;
+const RoofStatus = SDL.rpc.structs.RoofStatus;
+
+const Test = require('./../../../Test.js');
+const Validator = require('./../../../Validator.js');
+const BaseStructTests = require('./BaseStructTests');
+
+describe('RoofStatusTests', function () {
+    before(function () {
+        this.create = function () {
+            return new RoofStatus()
+                .setLocation(Test.GENERAL_GRID)
+                .setStatus(Test.GENERAL_DOORSTATUSTYPE)
+                .setState(Test.GENERAL_WINDOW_STATE);
+        };
+
+        this.getExpectedParameters = function (sdlVersion) {
+            return {
+                [RoofStatus.KEY_LOCATION]: Test.JSON_GRID,
```

Comment:
@ymalovanyi On line 19 does `Test.JSON_GRID` need to be changed to `Test.GENERAL_GRID`?

Reply:
Same as in above comment.